ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -21,70 +59,209 @@ interface Props {
+                  <li>
+                    the posting of my name and affiliation on a publicly accessible list of
+                     violators, and
+                  </li>
+                  <li>
+                    notification of the National Institute of Health or other federal agencies as
+                     to my actions.
+                  </li>
+                </ul>
+                <div style={{...styles.bold, ...styles.smallTopMargin}}>
+                  I understand that failure to comply with these requirements may also carry
+                   financial or legal repercussions. Any misuse of the <i>All of Us</i> Research
+                   Hub, Researcher Workbench, or the <i>All of Us</i> Research Program data is
+                   taken very seriously and other sanctions may be sought.
+                </div>
+                <label style={{...styles.bold, ...styles.largeTopMargin}}>Authorized Data user Name</label>
+                <DuaTextInput
+                    disabled
+                    data-test-id='ducc-username-input'
+                    value={profile.givenName + ' ' + profile.familyName}
+                />
+                <label style={{...styles.bold, ...styles.largeTopMargin}}>User ID</label>
+                <DuaTextInput
+                    disabled
+                    data-test-id='ducc-user-id-input'
+                    value={profile.username}
+                />
+                <label style={{...styles.bold, ...styles.largeTopMargin}}>Date</label>
+                <DuaTextInput type='text' disabled value={new Date().toLocaleDateString()}/>
+              </FlexColumn>
+              <FlexRow style={styles.dataUserCodeOfConductFooter}>
+                <Button
+                    type={'link'}
+                    style={{marginLeft: 'auto'}}
+                    onClick={() => this.setState({page: DataUserCodeOfConductPage.CONTENT})}
+                >
+                  Back
+                </Button>
+                <TooltipTrigger content={errors && <div>
+                  <div>All fields must be initialed</div>
+                  <div>All initials must match</div>
+                  <div>Initials must be six letters or fewer</div>
+                </div>}>
+                  <Button
+                      data-test-id={'submit-ducc-button'}
+                      disabled={errors || submitting}
+                      onClick={() => {
+                        this.submitDataUserCodeOfConduct(initialMonitoring);
+                      }}
+                  >
+                    Accept
+                  </Button>
+                </TooltipTrigger>
+              </FlexRow>
+            </React.Fragment>
+          }
+        </FlexColumn>;
+      } else {
+        return <div style={dataUserCodeOfConductStyles.dataUserCodeOfConductPage}>
+          <DataUseAgreementContentV1/>
+          <div style={{height: '1rem'}}/>
+          {getDataUseAgreementWidgetV1.call(this,
```

Comment:
Why invoke this via `call` rather than just invoking the function normally? If this is necessary, please add a comment to explain why.

Reply:
This is preexisting - this was what was in the render function before I started this implementation. I'd prefer not to spend effort refactoring code that we have an open ticket to delete.